Resources/datasets/market-hours/CryptoFuture.html

```diff
@@ -0,0 +1,8 @@
+<h4>BINANCE</h4>
+<table class="table qc-table table-reflow">
+<thead>
+<tr><th>Symbol</th><th>Contract</th></tr>
+</thead>
+<tbody>
```

Comment:
Is this table supposed to be empty?

Reply:
Yes. The converter will handle that when converting to HTML page